decidim-admin/app/permissions/decidim/admin/permissions.rb

```diff
@@ -3,6 +3,7 @@
 module Decidim
   module Admin
     class Permissions < Decidim::DefaultPermissions
+      include Decidim::UserRoleChecker
```

Comment:
I think this improves the legibility a bit

```suggestion
      include Decidim::UserRoleChecker

```

Reply:
And also make it consistent with the same include in other files